lib/ASRouterTriggerListeners.jsm

```diff
@@ -24,11 +24,28 @@ this.ASRouterTriggerListeners = new Map([
     _triggerHandler: null,
     _hosts: null,
 
+    /**
+     * Wait for browser startup to finish to avoid accessing uninitialized
+     * properties
+     */
+    async _checkStartupFinished(win) {
+      if (!win.gBrowserInit.delayedStartupFinished) {
+        await new Promise(resolve => {
```

Comment:
Some reason making the method async to await a single inline Promise without doing anything after doesn't seem quite right… but I guess it works ;)